src/themes.js

```diff
@@ -6,8 +6,8 @@ const colors = {
   green: '#00b777',
   grey: 'rgba(0, 0, 0, 0.2)',
   lightBlue: '#e6ebef',
-  lightPurple: '#f2eff7',
   offWhite: '#f7f9f6',
+  lightPurple: '#f2eff7',
```

Comment:
You don't need to change the order of these. I think I changed it in one of my latest PRs. I don't really understand how merging the master haven't solved this.

Reply:
i dont know how this occure. anyway i manually shift `lightPurple` before `offWhite`